README.md

```diff
@@ -0,0 +1,124 @@
+# celestia-da
+
+celestia-da is an implementation of the [Generic DA interface](https://github.com/rollkit/go-da)
+for modular blockchains. It extends celestia-node and runs a gRPC service,
+which can be used by rollup clients to read and write blob data to a specific
+namespace on celestia.
+
+Note that the rollup clients _do not_ need to depend on celestia, they can just
```

Comment:
this paragraph is more relevant for go-da readme. celestia-da is celestia specific. i would remove this.

Reply:
OK.